x/ecocredit/core/features/msg_retire.feature

```diff
@@ -127,4 +127,21 @@ Feature: MsgRetire
     }
     """
     When the message is validated
-    Then expect the error "invalid jurisdiction: foo, expected format <country-code>[-<region-code>[ <postal-code>]]: parse error"
+    Then expect the error "invalid jurisdiction: foo, expected format <country-code>[-<region-code>[ <postal-code>]]: parse error: invalid request"
+
+  Scenario: an error is returned if jurisdiction is an empty string
+    Given the message
+    """
+    {
+      "owner": "regen1depk54cuajgkzea6zpgkq36tnjwdzv4ak663u6",
+      "credits": [
+        {
+          "batch_denom": "C01-001-20200101-20210101-001",
+          "amount": "100"
+        }
+      ],
+      "jurisdiction": ""
+    }
+    """
+    When the message is validated
+    Then expect the error "jurisdiction cannot be empty, expected format <country-code>[-<region-code>[ <postal-code>]]: parse error: invalid request"
```

Comment:
This is a duplicate of `Scenario: an error is returned if jurisdiction is empty`. If jurisdiction is not provided, the zero-value is an empty string, which is why the error is the same here as it is above.

```suggestion
```

Reply:
Ah ok, I see. I have removed this.